content/docs/observability/deployment/offline.md

```diff
@@ -164,13 +166,39 @@ To perform an offline installation of a Helm chart, the following steps should b
     [user@anothersystem /home/user/offline-karavi-observability-bundle/helm]# kubectl get secret karavi-authorization-config proxy-server-root-certificate proxy-authz-tokens -n [CSI_DRIVER_NAMESPACE] -o yaml | sed 's/namespace: [CSI_DRIVER_NAMESPACE]/namespace: [CSM_NAMESPACE]/' | sed 's/name: karavi-authorization-config/name: isilon-karavi-authorization-config/' | sed 's/name: proxy-server-root-certificate/name: isilon-proxy-server-root-certificate/' | sed 's/name: proxy-authz-tokens/name: isilon-proxy-authz-tokens/' | kubectl create -f -
     ``` 
 
+    CSI Driver for PowerMax:   
```

Comment:
Does the offline installation need the same modification about copying configmap & secret?

Reply:
We cannot predict the customized name so I think no need to update offline installation, but that is better to add some comment in the script to explain offline script excepts default names.

Reply:
I will update csm-docs to add the comment to online installer, and for offline we can update the command.

Reply:
Updated. Please have a check.